src/PhpSpec/Formatter/ProgressFormatter.php

```diff
@@ -87,7 +87,7 @@ function ($count) use ($total) {
     private function getBarLengths($percents)
     {
         $stats = $this->getStatisticsCollector();
-        $specProgress = ($stats->getTotalSpecs()+1)/$stats->getTotalSpecsCount();
+        $specProgress = $stats->getTotalSpecsCount() ? ($stats->getTotalSpecs() + 1) / $stats->getTotalSpecsCount() : 0;
```

Comment:
http://en.wikipedia.org/wiki/Parkinson's_law_of_triviality

If nothing else, my error has brought @everzet back into the project ;-P